CIP-????/README.md

```diff
@@ -0,0 +1,474 @@
+---
+CIP: ????
+Title: Structured Contracts
+Category: Ledger/Tools
+Status: Active
+Authors:
+  - Polina Vinogradova <polina.vinogradova@iohk.io>
+Implementors: N/A
+Discussions:
```

Comment:
```suggestion
Discussions:
  - https://github.com/cardano-foundation/CIPs/pull/963
```
`this`

Reply:
fixed in https://github.com/cardano-foundation/CIPs/pull/963/commits/144c0181ac62c3a5fe8c350d4ba2f7c1e4a9056b